unlock-app/src/actions/error.ts

```diff
@@ -0,0 +1,19 @@
+import { UnlockError } from '../utils/Error' // eslint-disable-line
+
+export const SET_ERROR = 'error/SET_ERROR'
+export const RESET_ERROR = 'error/RESET_ERROR'
+export const CLEAR_ALL_ERRORS = 'error/CLEAR_ALL_ERRORS'
+
+export const setError = (error: UnlockError) => ({
+  type: SET_ERROR,
+  error,
+})
+
+export const resetError = (error: UnlockError) => ({
+  type: RESET_ERROR,
+  error,
+})
+
+export const clearAllErrors = () => ({
```

Comment:
This is new -- we used to have some behavior that cleared all errors if `resetError` was called with an undefined argument. I'd rather this behavior (so far unused) be explicit, so I've introduced a new action.